crates/subspace-networking/src/connected_peers.rs

```diff
@@ -219,41 +219,59 @@ impl<Instance> Behaviour<Instance> {
 
     /// Specifies the whether we should keep connections to the peer alive. The decision could
     /// depend on another protocol (e.g.: PeerInfo protocol event handling).
-    /// In case when we had decision timeout it sets up proper keep connection alive anyway.
     pub fn update_keep_alive_status(&mut self, peer_id: PeerId, keep_alive: bool) {
-        let (connection_state, keep_alive) = if keep_alive {
-            if self.permanently_connected_peers() < self.config.target_connected_peers {
-                trace!(%peer_id, %keep_alive, "Insufficient number of connected peers.");
+        let enough_connected_peers =
+            self.permanently_connected_peers() < self.config.target_connected_peers;
```

Comment:
Shouldn't the condition be inverted?:
```suggestion
            self.permanently_connected_peers() >= self.config.target_connected_peers;
```

This below is confusing because of it, I do not understand why you called it the way you did, makes no sense to me:
```
if enough_connected_peers {
    trace!(%peer_id, %keep_alive, "Insufficient number of connected peers.");
}
```

Reply:
It's a confusing name indeed. I added a "not_" prefix to the variable name!